docs/RELEASING.md

```diff
@@ -0,0 +1,7 @@
+# Releasing
+
+## Bumping version
+
+To change the version the following steps have to be executed:
+* Update the version number in [TracerVersion.cs](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/tools/Datadog.Core.Tools/TracerVersion.cs).
+* Run the [PrepareRelease tool](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/tree/main/build/tools/PrepareRelease): `dotnet run versions integrations`.
```

Comment:
```suggestion
* Run the [PrepareRelease tool](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/tree/main/build/tools/PrepareRelease): `dotnet run msi versions integrations`.
```

Using the `msi` argument is somewhat unrelated to "bumping version" but it updates the dll's and pdb's included in the MSI and must be done before releasing